jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+          example: NCVV
+          maxLength: 10
+        type:
+          type: string
+          x-stoplight:
+            id: isvadpbkeknvt
+          example: CONT
+          description: "Categorization of ocean-going vessels distinguished by the main cargo the vessel carries. Possible values:\r\n\r\nGCGO (General cargo)\r\n- CONT (Container)\r\n- RORO (RoRo)\r\n- CARC (Car carrier)\r\n- PASS (Passenger)\r\n- FERY (Ferry)\r\n- BULK (Bulk)\r\n- TANK (Tanker)\r\n- LGTK (Liquefied gas tanker)\r\n- ASSI (Assistance)\r\n- PILO (Pilot boat)"
+        draft:
+          type: number
+          x-stoplight:
+            id: vmtb7rovvvypo
+          format: float
+          example: 12.5
+          description: "The actual draft of the vessel.\r\n\r\nIf the draft is specified in feet (`FOT`) then the decimal part should be concidered as a fraction of a foot and not as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6\")."
+        dimensionUnit:
+          type: string
+          x-stoplight:
+            id: ts5uqfhvusv3h
+          example: MTR
+          description: "The unit of measure which can be expressed in\r\n\r\n- MTR (Meter)\r\n- FOT (Foot)"
+    GeoLocation:
+      title: GeoLocation
+      x-stoplight:
+        id: el0ns1p97ns6h
+      type: object
+      required:
+        - latitude
+        - longitude
+      properties:
+        locationName:
+          type: string
+          x-stoplight:
+            id: o9bpp2tncry7j
+          description: The name of the location.
+          example: Port of Amsterdam
+          maxLength: 100
+        latitude:
+          type: string
+          x-stoplight:
+            id: wejtvnxsenx3w
+          description: Geographic coordinate that specifies the north–south position of a point on the Earth's surface.
+          example: '48.8585500'
+          maxLength: 10
+        longitude:
+          type: string
+          x-stoplight:
+            id: rde1j4blorw0x
+          description: Geographic coordinate that specifies the east–west position of a point on the Earth's surface.
+          maxLength: 11
+          example: '2.294492036'
+      description: An interface used to express a location using `latitude` and `longitude`.
+    AddressLocation:
+      title: AddressLocation
+      x-stoplight:
+        id: aqjpjncpt0gbe
+      type: object
+      required:
+        - address
+      properties:
+        locationName:
+          type: string
+          x-stoplight:
+            id: 9nyqb9e5709af
+          description: The name of the location.
+          maxLength: 100
+          example: Port of Amsterdam
+        address:
```

Comment:
Does this differ from the Address object in line 239? If no - should it be a $ref instead? What do you think? (In the future - Spectral will catch if there are changes. It is ok for me if you lease as is...

Reply:
Yes it is the same. Changed to $ref